src/o365/spo/commands/hubsite/hubsite-list.ts

```diff
@@ -22,7 +30,20 @@ class SpoHubSiteListCommand extends SpoCommand {
     return 'Lists hub sites in the current tenant';
   }
 
+  constructor() {
+    super()/* istanbul ignore next */;
+    this.batchSize = 30;
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.classifications = args.options.includeAssociatedSites === true;
```

Comment:
The telemetry property doesn't match the option. It should be `includeAssociatedSites` instead